code/game/objects/items/storage/sixpack.dm

```diff
@@ -34,6 +34,41 @@
 	name = "beer can ring"
 	desc = "Holds six beers. Remember to recycle when you're done!"
 
+/obj/item/storage/cans/sixgamerdrink
+	name = "gamer drink bottle ring"
+	desc = "Holds six gamer drink cans. Remember to recycle when you're done!"
+
+	/// Pool of gamer drinks tm we may add from
+	var/list/gamer_drink_options = list(
+		/obj/item/reagent_containers/cup/soda_cans/pwr_game = 55,
+		/obj/item/reagent_containers/cup/soda_cans/space_mountain_wind = 15,
+		/obj/item/reagent_containers/cup/soda_cans/monkey_energy = 15,
+		/obj/item/reagent_containers/cup/soda_cans/volt_energy = 10,
+		/obj/item/reagent_containers/cup/soda_cans/thirteenloko = 5,
+	)
+
+/obj/item/storage/cans/sixgamerdrink/PopulateContents()
+	for(var/i in 1 to 6)
+		var/obj/item/chosen_gamer_drink = pick_weight(gamer_drink_options)
+		new chosen_gamer_drink(src)
+
 /obj/item/storage/cans/sixbeer/PopulateContents()
 	for(var/i in 1 to 6)
 		new /obj/item/reagent_containers/cup/soda_cans/beer(src)
```

Comment:
Looks like you put `/obj/item/storage/cans/sixgamerdrink` in between `/obj/item/storage/cans/sixbeer` and `/obj/item/storage/cans/sixbeer/PopulateContents()`